.circleci/config.yml

```diff
@@ -75,16 +77,18 @@ executors:
       YARN_ENABLE_GLOBAL_CACHE: true
       # Enabling hard links saves about 1.2 GB of space in the image.
       YARN_NM_MODE: hardlinks-global
+      HUSKY_SKIP_INSTALL: 1
 
   # Contains prebuilt image with low install overhead. Perfect for the first build
   # step, and the saving the initial a workspace state.
   build-executor:
     docker:
-      - image: mozilla/fxa-circleci:ci-builder
+      - image: mozilla/fxa-circleci:ci-builder-nx
```

Comment:
Oh ok I think I get what's going on.  Ignore my previous comment.  😅

Reply:
See comment above, this was a bit confusing.